experimenter/tests/integration/nimbus/utils/helpers.py

```diff
@@ -13,10 +14,13 @@
 
 
 def load_graphql_data(query):
+    nginx_url = "https://nginx"
+    if not os.environ.get("DEBIAN_FRONTEND"):
+        nginx_url = "https://localhost"
```

Comment:
Same here with a different variable name, INTEGRATION_TEST_NGINX_URL or something

Reply:
Not a bad idea, it would be an extra step the user would need to do though. I don't think tox can do the wildcard we have now and another variable.

Reply:
what do you mean? `passenv = *` will pass *all* variables through